src/main/java/org/jitsi/impl/reservation/rest/RESTReservations.java

```diff
@@ -217,7 +216,7 @@ else if (result.statusCode == 409
     /**
      * Deletes conference for given <tt>mucRoomName</tt> through the API.
      */
-    Result deleteConference(String mucRoomName)
+    private synchronized Result deleteConference(String mucRoomName)
```

Comment:
It is not "incidental" if it was written this way in purpose to take care of access synchronisation in the public methods, but I agree it may not be readable enough.